src/sentry/static/sentry/app/components/asyncComponentSearchInput.tsx

```diff
@@ -158,11 +158,4 @@ const Form = styled('form')`
   position: relative;
 `;
 
-// XXX(epurkhiser): The withRouter HoC has incorrect typings. It does not
-// correctly remove the WithRouterProps from the return type of the HoC, thus
-// we manually have to do this.
-type PropsWithoutRouter = Omit<Props, keyof ReactRouter.WithRouterProps>;
```

Comment:
Is there another place we do this? or was this the only one you found? I thought there may have have been another place I copied this? Maybe getsentry?

Reply:
Ah there's another it looks like, thanks